linera-explorer/src/input_type.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use super::js_utils::{getf, unproxy};
+use wasm_bindgen::prelude::*;
+
+/// add a input line for lists
```

Comment:
All method and function documentation should start with uppercase, and in third person singular. In general, if in doubt, let's follow the standard library's example.

```suggestion
/// Adds an input line for lists.
```